storm-server/src/test/java/org/apache/storm/daemon/nimbus/NimbusTest.java

```diff
@@ -24,8 +24,10 @@
 import org.apache.storm.topology.TopologyBuilder;
 import org.junit.Test;
 
+import static org.junit.Assert.*;
```

Comment:
Checkstyle does not check tests for some reason, but if you care about it I can change it.